### PR TITLE
[De-ja-vu edition] Dream weapon skills, sew/weaving re-naming/organizing for both players and coders.

### PR DESCRIPTION
**Pull request by Red-Honk**
[CHANGELOGS]:
Removing sell prices, if one wishes to sell these things, just scavenge them for parts and sell that, from sew/weave products.
Exempt lordly cloak and coat silks for they are luxury items, for the vault or merchant to sell. (sticky fingers)
Kingsfield decrees, "All holy items are now barred from vault scrying and merchant guild selling."

[Added dreams]: (Acting like mini-tutorials that I'll refine with time)

Dream skill: [Archery]: "...you stand in an open field, a straw target in your sights, pulling a arrow from your quiver with your gloved hand and loading it to aim and let it loose at the target missing, this cycle continued for a bit until your aim steadied proper, muscles building up strength making it easier to draw the bow, skills slowly refining aiding the task at hand, a new trick that pop to mind is to curl the arm back and grab the arrow with the bow holding hand to draw it in the air before aiming proper, tilting the head, squinting an eye, taking a deep breath and finally letting it loose, bullseye, probably want to run just about now, that bull looks quite angry..."

Dream skill: [Axe]: "...grabbing a random axe, using it to cut and chop, getting a feel for it and finally realizing that they are not half bad one handed with barely any added heft when wielded, be it iron or steel, until picking up the battle axe and the effect inverted, what a cumbersome weapon to wield in one-hand, with a mighty fine one in two, chopping through limps like, shattering shields into splinters and bending steel one to ones will, reaching the legs, groin and above, what a handy thing to axe folk a few questions..." (Note: this was written in haste to push this thing in already)

Dream skill: [Fist-fighting]: "...your body is bruised, your breathe haggers, but your harden fists and lower-legs refused to give in, going into actions with punches that landed furied blows from their groin to the top that managed to fractured several bones before dealing a decisive knockout that was followed up by a hefty punch that snapped the spine, refocusing your efforts towards another, deciding to use an delayed kick against the unarmored leg, avoiding the neck and above that can't be reached, knocking one by the wall, another on the table, a third against a few objects and a fourth against fifth, moving into a stomp that dislocated and later shattered bone, you stood there, gasping for air, all bruised and beaten up, yet victories..."

Dream skill: [Polearms]: "...the air blows against your body in the open field, as your opponent's whole body is at range with the polearm's great length, regaining your focus you'd start to pull the polearm with a firm grip to ready up for a proper instead of a half-hearted hasty half-forced strike and thrusting it forth, keeping your opponent at a distance, striking where they move or at when close and with proper precision when they're at range, with double the said range came the advantage when chasing and tactically falling back from them, piercing through padded-cloth and leather armor like they were nothing with the spear, throwing it towards someone unarmored before rolling towards an heavier polearm and reaching for the heavens and with the added polearm range unleashed a slow but crashing blow through that plated armor making them scream for pain and slowly crawl towards and away, never being able to reach you back until they were properly subdued...""

Dream skill: [Knife-fighting]: "...your heart beats as swift as your knife strikes that reached their legs, groin and above, throwing daggers and in a devious move kicking them from the floor towards ones that tried to dodge and parry to catch them by surprise, at times when timed right with attacking them directly go through as they recover their parry, draining chipping slowly at their armor and making them slowly short of breathe, while dodging which made it even worse and feinting their parry attempts to begin unleashing a fury of stabs through their leather armor before flipping the knife over to start picking through chained and plated armor alike, fully ignoring its protection opponent's surface, with great haste and ease of precision little stood chance against your swift devious might..."

Dream skill: [Mace]: "...maces maces maces, they're all around, striking to slow, unleashing a hefty, delayed smash to fully punch though its protection, stunning them briefly and staggering them after, making their retreats and advances more than futile, smashing them when they're down, making them too meek to stand on their legs, what an amusing sight." (Note: this was written in haste to push this thing in already)

[Sewing/weaving]: Can now be easily more accessible without the wiki, as in, by pressing the first letter that comes to mind say an item such as a cloak by tapping [C], item (padded/material/style/ect) [(ingredient number if more than one), ingredient a (with a s if is more than one), ingredient b (if any); (Skill)

Other words:
Cloak (2 cloths, fiber; APPRENTICE)
Cloak (half, silk) - (2 cloths, silk; JOURNEYMAN)

Why It's Good For The Game
Dreams as mini-tutorials are a smart way to teach in an IC way, even if not fully baked.
Making crafting easier to find what one needs and how much it costs and what skill level it requires is a very helpful way to purge pointless searching and wasting time with the tap of the first letter of the item in question.
Code being more organized in an Alphabetical order, makes it way easier to code from.

### CODE REVIEW

**Comment by Nakalon** [comment]
I'm still of the opinion these dreams are excessively verbose! I liked the short versions, in honesty if a text this size came up in game I would not read it!

**Comment by Nakalon** [comment]
Also you should mention what sell prices you are removing! As it stands it seems you're removing them ALL when that's not true!

**Comment by Red-Honk** [comment]
> I'm still of the opinion these dreams are excessively verbose! I liked the short versions, in honesty if a text this size came up in game I would not read it!

The ones before it was simply flavor text, this one is informative flavor text, you can either read it or not, it makes little difference.

> Also you should mention what sell prices you are removing! As it stands it seems you're removing them ALL when that's not true!

It quite literally does in changelogs: Removing sell prices, if one wishes to sell these things, just scavenge them for parts and sell that, >>>>>>.from sew/weave products.<<<<<<

**Comment by Nakalon** [comment]
> > Also you should mention what sell prices you are removing! As it stands it seems you're removing them ALL when that's not true!
> 
> It quite literally does in changelogs: Removing sell prices, if one wishes to sell these things, just scavenge them for parts and sell that, >>>>>>.from sew/weave products.<<<<<<

Don't fault me you know I slept like 3 hours 😭

File: code/datums/skills/combat.dm [hunk 1]
```diff
@@ -50,24 +53,48 @@
...
+	)
 
 /datum/skill/combat/maces
 	name = "Maces"
+	dreams = list(
+		"...maces maces maces, they're all around, striking to slow, unleashing a hefty delayed smash to fully penetrate all that would rather stand in your path, stunning them briefly, smashing them when they're down to be unable to get up, what an amusing sight."
```
Comment (Cyprex) [inline code/datums/skills/combat.dm:63]: Maces aren't about penetrating, they're about blunt trauma, no? As I understand they're an anti-armor weapon meant to deliver a lot of force, denting armor is a viable strategy- damaged armor can restrict the wearers range of motion, or even injure them.

Comment (Red-Honk) [reply code/datums/skills/combat.dm:63]: They quite literally have armor penetration 80 on smash, same as the pick intent on daggers, so, yes, they pierce armor hard, the majority of armors in the game, have not enough points to absorb the full brunt of a mace smash, you may as well be wearing nothing, for plate can't stop it from dealing full damage, the only thing it stops is getting critted by it.

**Comment by katsukai** [comment]
Heres one for unarmed.

headbutt. headbutt. headbutt. headbutt is the fastest, it is faster than anything. it is faster than light, it is more than we can comprehend. it is a beam of pure thought directed from my forehead into yours at a billion miles a second.

i headbutted someone today and when the smoke cleared we were both 10 years younger. he tried to feint me but before his brain could even send the impulse to the nerves in his body, telling him to point over my shoulder, i had headbutted him twice. sometimes i do it so quickly i don't even know it happened, i am just instantly surrounded by mountains of bodies, heads crushed like old fruit and cocoons undeployed.

**Comment by Nakalon** [comment]
> Heres one for unarmed.
> 
> headbutt. headbutt. headbutt. headbutt is the fastest, it is faster than anything. it is faster than light, it is more than we can comprehend. it is a beam of pure thought directed from my forehead into yours
> 
> i headbutted someone today and when the smoke cleared we were both 10 years younger. he tried to feint me but before his brain could even send the impulse to the nerves in his body, telling him to point over my shoulder, i had headbutted him twice. sometimes i do it so quickly i don't even know it happened, i am just instantly surrounded by mountains of bodies, heads crushed like old fruit and cocoons undeployed.

LMAO

**Pull request merged**